ansible/configs/cloud-credentials/README.adoc

```diff
@@ -0,0 +1,26 @@
+== Overview
+
+*cloud-credentials* _config_ is a simple config that will provision and make available as a variable a set of cloud credentials.
+This exists to leverage the same overall workflow, but avoid creating infrastructure that is not needed.
+
+== Supported Cloud Providers
+
+Currently, only the following cloud providers are supported.
+For this to be useful, a sandbox type mechanism is required so that the credential provisioned is isolated from anything else.
+
+* ec2
+
+== AgnosticV Requirements
+
+In your agnosticV config, you should have the following set to enable provisioning of the sandboxy account
+
+----
+agnosticv_meta:
+  sandboxed: true
+----
```

Comment:
Let's add the equivalent for Babylon:

```yaml
__meta__:
  aws_sandboxed: true
```


Also, make sure we set the variable to force aws-nuke.

```yaml
agnosticv_meta:
  sandboxed: true
  nuke_sandbox: true
```

In babylon, all sandboxes are marked for cleanup (with aws-nuke)  see https://github.com/redhat-gpte-devopsautomation/babylon_aws_sandbox/blob/master/tasks/release.yaml#L47.